fml/raster_thread_merger.cc

```diff
@@ -17,23 +17,37 @@ RasterThreadMerger::RasterThreadMerger(fml::TaskQueueId platform_queue_id,
       gpu_queue_id_(gpu_queue_id),
       task_queues_(fml::MessageLoopTaskQueues::GetInstance()),
       lease_term_(kLeaseNotSet) {
-  is_merged_ = task_queues_->Owns(platform_queue_id_, gpu_queue_id_);
+  bool is_merged = task_queues_->Owns(platform_queue_id_, gpu_queue_id_);
```

Comment:
@iskakaushik I saw that in the current code, we are expecting that the threads might be merged before constructing `RasterThreadMerger`. Do you recall in what case that could be?
I feel like we should ensure that the threads are un-mergerd when constructing the `RasterThreadMerger`.